src/Hl7.Fhir.Core/Rest/FhirClientSearch.cs

```diff
@@ -604,14 +739,31 @@ private SearchParams toQuery(string[] criteria, string[] includes, int? pageSize
             return q;
         }
 
-        #endregion
+        private (string path, IncludeModifier? modifier)[] stringToIncludeTuple(string[] includes)
+        {
+            if(includes != null && includes.Any())
+            {
+                var modifierIncludes = new List<(string, IncludeModifier?)>();
+                foreach (var i in includes)
+                {
+                    modifierIncludes.Add((i, null));
+                }
+                return modifierIncludes.ToArray();
```

Comment:
includes.Select(i => (i, (IncludeModifier?)null)).ToArray()?

Reply:
and even better (re: our discussion):   `includes.Select(i => (i, IncludeModifier.None)).ToArray()`.

Reply:
Done!